src/dmd/dsymbolsem.d

```diff
@@ -555,6 +555,87 @@ private uint setMangleOverride(Dsymbol s, const(char)[] sym)
     return 0;
 }
 
+/*
+  Compiles the mixin content of CompileExp, CompileDeclaration
+  and CompileStatement.
+
+  Params:
+    CompileEntity = the type of mixin
+    sc            = the scope containing the mixin
+
+  Returs:
+    The resulting AST after the parsing phase
+*/
+auto compileIt(CompileEntity)(CompileEntity ce, Scope* sc)
```

Comment:
Making this a template is overkill

Reply:
Thats kind of the whole point of this PR